src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/SqlClientMetaDataCollectionNames.cs

```diff
@@ -0,0 +1,53 @@
+
+        /// <include file='../../../../../../doc/snippets/Microsoft.Data.SqlClient/SqlClientMetaDataCollectionNames.xml' path='docs/members[@name="SqlClientMetaDataCollectionNames"]/ForeignKeys/*' />
+        public static readonly string ForeignKeys = "ForeignKeys";
+
+        /// <include file='../../../../../../doc/snippets/Microsoft.Data.SqlClient/SqlClientMetaDataCollectionNames.xml' path='docs/members[@name="SqlClientMetaDataCollectionNames"]/IndexColumns/*' />
+        public static readonly string IndexColumns = "IndexColumns";
+
+        /// <include file='../../../../../../doc/snippets/Microsoft.Data.SqlClient/SqlClientMetaDataCollectionNames.xml' path='docs/members[@name="SqlClientMetaDataCollectionNames"]/Indexes/*' />
+        public static readonly string Indexes = "Indexes";
+
+        /// <include file='../../../../../../doc/snippets/Microsoft.Data.SqlClient/SqlClientMetaDataCollectionNames.xml' path='docs/members[@name="SqlClientMetaDataCollectionNames"]/ProcedureParameters/*' />
+        public static readonly string ProcedureParameters = "ProcedureParameters";
+
+        /// <include file='../../../../../../doc/snippets/Microsoft.Data.SqlClient/SqlClientMetaDataCollectionNames.xml' path='docs/members[@name="SqlClientMetaDataCollectionNames"]/Procedures/*' />
+        public static readonly string Procedures = "Procedures";
+
+        /// <include file='../../../../../../doc/snippets/Microsoft.Data.SqlClient/SqlClientMetaDataCollectionNames.xml' path='docs/members[@name="SqlClientMetaDataCollectionNames"]/Tables/*' />
+        public static readonly string Tables = "Tables";
+
+        /// <include file='../../../../../../doc/snippets/Microsoft.Data.SqlClient/SqlClientMetaDataCollectionNames.xml' path='docs/members[@name="SqlClientMetaDataCollectionNames"]/UserDefinedTypes/*' />
+        public static readonly string UserDefinedTypes = "UserDefinedTypes";
+
+        /// <include file='../../../../../../doc/snippets/Microsoft.Data.SqlClient/SqlClientMetaDataCollectionNames.xml' path='docs/members[@name="SqlClientMetaDataCollectionNames"]/Users/*' />
+        public static readonly string Users = "Users";
+
+        /// <include file='../../../../../../doc/snippets/Microsoft.Data.SqlClient/SqlClientMetaDataCollectionNames.xml' path='docs/members[@name="SqlClientMetaDataCollectionNames"]/ViewColumns/*' />
+        public static readonly string ViewColumns = "ViewColumns";
+
+        /// <include file='../../../../../../doc/snippets/Microsoft.Data.SqlClient/SqlClientMetaDataCollectionNames.xml' path='docs/members[@name="SqlClientMetaDataCollectionNames"]/Views/*' />
+        public static readonly string Views = "Views";
+
+        /// <include file='../../../../../../doc/snippets/Microsoft.Data.SqlClient/SqlClientMetaDataCollectionNames.xml' path='docs/members[@name="SqlClientMetaDataCollectionNames"]/AllColumns/*' />
+        public static readonly string AllColumns = "AllColumns";  // supported starting from SQL Server 2008
```

Comment:
Minimum SQL Server supported is SQL Server 2012, comment no longer applicable.
```suggestion
        public static readonly string AllColumns = "AllColumns";
```